pages/webhosting/reference-content/classic-hosting-migration-information.mdx

```diff
@@ -30,7 +30,7 @@ You can initiate the migration of your Web Hosting plan directly from your [Dedi
 
 ### What will my new hosting plan be?
 
-After [initiating the migration](/webhosting/how-to/migrate-webhosting-plan/) of your Online.net hosting plan, it will be transferred to Scaleway. During this transition, your existing offer will be temporarily moved to a migration plan. You will then have the possibility to select from Scaleway's range of Web Hosting plans, tailored to meet your specific requirements.
+After [initiating the migration](/webhosting/how-to/migrate-webhosting-plan/) of your Online.net hosting plan, it will be transferred to Scaleway. During this transition, your existing offer will be temporarily moved to a migration plan. You will then have the possibility to select a Scaleway Web Hosting plan, tailored to meet your specific requirements.
```

Comment:
```suggestion
After [initiating the migration](/webhosting/how-to/migrate-webhosting-plan/) of your Online.net hosting plan, it will be transferred to Scaleway. During this transition, your existing offer will be temporarily moved to a migration plan. You can then select a Scaleway Web Hosting plan, tailored to meet your specific requirements.
```